app/src/pages/index.tsx

```diff
@@ -21,21 +25,15 @@ export default function Home(props: { users: User[] }) {
       <main className={styles.main}>
         <h1 className={styles.title}>Welcome</h1>
 
-        <button>{'<<'}</button>
+        {props.page != 0 && <button>{<Link href={`http://localhost:3000/?page=${props.page - 1}`}>{'<<'}</Link>}</button>}
```

Comment:
```suggestion
        {props.page != 0 && (
          <button>
            {
              <Link
                href={{
                  pathname: "/",
                  query: { page: props.page - 1 },
                }}
              >
                {"<<"}
              </Link>
            }
          </button>
        )}
```
実はNext.jsの`Link`は高性能でこんな感じでかくとquery paramを簡単に設定できるので、こうやるのが良い手です！！